src/settings-ui/Settings.UI/SettingsXAML/Views/CmdNotFoundPage.xaml

```diff
@@ -19,37 +26,86 @@
+                                Symbol="Cancel"
+                                Visibility="{x:Bind Mode=OneWay, Path=ViewModel.IsWinGetClientModuleDetected, Converter={StaticResource BoolToInvertedVisibilityConverter}}" />
+                            <Button
+                                x:Uid="CmdNotFound_InstallButton"
+                                Command="{x:Bind ViewModel.InstallWinGetClientModuleEventHandler}"
+                                IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.IsWinGetClientModuleDetected, Converter={StaticResource BoolNegationConverter}}"
+                                Style="{StaticResource AccentButtonStyle}" />
+                        </StackPanel>
+                    </controls:SettingsCard>
+                </custom:SettingsGroup>
+                <custom:SettingsGroup x:Uid="CmdNotFound_InstallationGroup" IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.IsEnabledGpoConfigured, Converter={StaticResource BoolNegationConverter}}">
+                    <controls:SettingsCard
+                        x:Uid="CmdNotFound_Enable"
+                        HeaderIcon="{ui:BitmapIcon Source=/Assets/Settings/FluentIcons/FluentIconsCmdNotFound.png}"
+                        IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.IsEnabledGpoConfigured, Converter={StaticResource BoolNegationConverter}}">
+                        <StackPanel Orientation="Horizontal" Spacing="8">
+                            <SymbolIcon
+                                Foreground="{ThemeResource SystemFillColorSuccessBrush}"
+                                Symbol="Accept"
+                                Visibility="{x:Bind Mode=OneWay, Path=ViewModel.IsCommandNotFoundModuleInstalled, Converter={StaticResource BoolToVisibilityConverter}}" />
+                            <SymbolIcon
+                                Foreground="{ThemeResource SystemFillColorCriticalBrush}"
+                                Symbol="Cancel"
+                                Visibility="{x:Bind Mode=OneWay, Path=ViewModel.IsCommandNotFoundModuleInstalled, Converter={StaticResource BoolToInvertedVisibilityConverter}}" />
+                            <Button
+                                x:Uid="CmdNotFound_InstallButton"
+                                Command="{x:Bind ViewModel.InstallModuleEventHandler}"
+                                Style="{StaticResource AccentButtonStyle}"
+                                Visibility="{x:Bind Mode=OneWay, Path=ViewModel.IsCommandNotFoundModuleInstalled, Converter={StaticResource BoolToInvertedVisibilityConverter}}" />
+                            <HyperlinkButton
+                                x:Uid="CmdNotFound_UninstallButton"
+                                Command="{x:Bind ViewModel.UninstallModuleEventHandler}"
+                                Visibility="{x:Bind Mode=OneWay, Path=ViewModel.IsCommandNotFoundModuleInstalled, Converter={StaticResource BoolToVisibilityConverter}}" />
```

Comment:
As we now hide we should have the same style like the install button.

Reply:
@niels9001 currently doing some style fixes :)